config/postcreate-hooks/kubestellar.yaml

```diff
@@ -189,6 +189,23 @@ spec:
               - "ControlPlaneName={{.ControlPlaneName}}"
             env:
             - name: XDG_CACHE_HOME
-              value: /tmp/helm/.cache  
+              value: /tmp/helm/.cache
+          - name: "{{.HookName}}-otp"
+            image: quay.io/kubestellar/helm:3.14.0
+            imagePullPolicy: IfNotPresent
+            args:
+              - upgrade
+              - --install
+              - --namespace
+              - "{{.Namespace}}"
+              - ocm-transport-plugin
+              - oci://ghcr.io/kubestellar/ocm-transport-plugin/chart/ocm-transport-plugin
+              - --version
+              - "0.1.7"
```

Comment:
Could we use the PCH support for user-defined objects (or vars) to avoid having these versions numbers hardcoded in the PCH ? See https://github.com/kubestellar/kubeflex/blob/main/docs/users.md#user-provided-objects

Reply:
@pdettori I could do that way, and I'm fine changing it. However, somewhat, I think we want to hide the specific dependencies to the general user similar to what you do for postgress.
The current way, one specify the version of KS and the PCH already use the correct dependencies.
In your proposed way, one would have to specify the version of KS PCH, then specify the version of status addon and otp on the kflex command line... more flexible, I guess but more prone to errors.

@MikeSpreitzer @ezrasilvera do you also have an opinion?
I'm fine either way

BTW, this will be superseded by the hub-chart where my understanding of the feedback was to hard code the dependencies in the yaml and not expose them in the `values.yaml` .

Reply:
I am aware of the desire of hiding deps and I agree. I was more concerned with how in many places we need to update dependencies when we make a new release. It would be an improvement to find ways to define all versions only in one single place rather than scattered all over our code.

Reply:
The anticipated hub chart will be a convenient all-in-one bundle but not the only way to create a WDS. I do not think that we should require that all the WDSes are created up-front. I would not want to make the user creating a WDS some time after the initial setup have to specify the ks/OTP release to use.

Reply:
.. and when a user later adds a WDS, we should not make them specify the OTP version. That is why I approve of specifying the OTP version here.